docs/docs/actions.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: Actions
-description: Documentation on actions and how they help you manipulate state within Gatsby
+description: Dokumentation für Actions und wie diese helfen State in Gatsby zu manipulieren
```

Comment:
```suggestion
description: Dokumentation für Actions und wie sie dir helfen State in Gatsby zu manipulieren
```

"du" is used on the other pages aswell :)

Reply:
I would argue, that this doesn't need to be personal here, alternatively I'd prefer `diese dir` instead of `sie dir`. What do you think?

Reply:
I agree it doesn't need to be personal and somewhere in a paragraph I wouldn't have changed it, but because it's a headline which leads you through the whole thing and in the other tutorials they were all written in this personal way, I'd argue to make it personal here aswell.

I think both works - the personal one maybe a bit better, but no strong feelings :) Maybe someone else has opinions too?

Reply:
I think this is a very minor detail. I'll commit your suggestion!